contracts/governance/GoodDollarStaking.sol

```diff
@@ -234,27 +234,52 @@ contract GoodDollarStaking is
 		_setAPY(_interestRatePerBlock);
 	}
 
-	/// @dev helper function for multibase
-	function getRewardsPerBlock() public view returns (uint256) {
-		return rewardsPerBlock[address(this)];
+	/// @dev helper function for multibase and FixedAPYRewards
+	function getRewardsPerBlock() public view returns (uint256 _goodRewardPerBlock, int128 _gdRewardPerBlockX64) {
+		_goodRewardPerBlock = rewardsPerBlock[address(this)];
+    _gdRewardPerBlockX64 = interestRatePerBlockX64;
 	}
 
-	/// @dev helper function for multibase
+	/// @dev helper function for multibase and FixedAPYRewards - same stake amount for both
 	function getStaked(address _user) public view returns (uint256, uint256) {
 		return getProductivity(address(this), _user);
 	}
 
-	/// @dev helper function for multibase
-	function getUserPendingReward(address _user) public view returns (uint256) {
-		return getUserPendingReward(address(this), 0, block.number, _user);
+  /// @dev helper function for multibase and FixedAPYRewards
+	function getUserPendingReward(address _user)
+		public
+		returns (uint256 _goodReward, uint128 _gdReward)
+	{
+		// GOOD rewards
+    _goodReward = getUserPendingReward(
+			address(this),
+			0,
+			block.number,
+			_user
+		);
+    
+    // g$ rewards
+    _updateReward(_user);
+    _gdReward = stakersInfo[_user].reward;
+	}
+
+	/// @dev helper function for multibase and FixedAPYRewards
+	function totalRewardsPerShare()
+		public
+		view
+		returns (uint256 _goodRewardPerShare, uint128 _gdRewardPerShare)
+	{
+		_goodRewardPerShare = super.totalRewardsPerShare(address(this));
+		_gdRewardPerShare = rewardPerTokenStored;
 	}
 
-	/// @dev helper function for multibase
+  /// @dev helper function for multibase
```

Comment:
whats going on with the indentations? fix your editor to automatically indent and prettier on save

Reply:
I don't know, sometimes it adds spaces although it doesn't look like that locally.
Added format on save and will fix it for good